src/ChainRules.jl

```diff
@@ -2,6 +2,7 @@ module ChainRules
 
 using Reexport
 @reexport using ChainRulesCore
```

Comment:
Why not go all the way and reexport everything by hand and then we can drop the dependency on Reexport.jl?

Reply:
Because if we add an export to ChainRulesCore we also have to modify ChainRules. And Reexport is a one-file package with no dependencies.